File: 19/Go/signature.go
package main

import (
	"crypto/hmac"
	"crypto/sha1"
	"encoding/base64"
	"encoding/hex"
	"fmt"
)

func generateSignature(data string) string {
	key, err := hex.DecodeString("DFA5ED192DDA6E88A12FE12130DC6206B1251E44")
	if err != nil {
		fmt.Println("Error: ", err)
	}

	mac := hmac.New(sha1.New, key)
	mac.Write([]byte(data))
	signature := append([]byte{0x19}, mac.Sum(nil)...)
	return base64.StdEncoding.EncodeToString(signature)
}

func main() {
	fmt.Println(generateSignature("teste"))
}

File: 19/Python/signature.py
import hmac
import base64
import json
import time
from hashlib import sha1


def generate_signature(data: str) -> str:
    mac = hmac.new(bytes.fromhex("DFA5ED192DDA6E88A12FE12130DC6206B1251E44"), data.encode(), sha1)
    return base64.b64encode(b"\x19" + mac.digest()).decode("utf-8")


payload = json.dumps({
    "timestamp": int(time.time() * 1000)
})

print(generate_signature(payload))